ui/.babelrc

```diff
@@ -18,21 +18,21 @@
         "development": {
             "plugins": [
                 ["babel-plugin-provide-modules", {
-                    "utils/h.jsx": "h"
+                    "@guardian/guui": ["h"]
```

Comment:
would it be clearer to make these exports be individual files? so they were `@guardian/guui/h`, `@guardian/guui/svg-loader` etc?

Reply:
Thanks for reviewing, @sndrs. There's a lot to unpack here.

`@guardian/guui` currently has three exported files:

- `@guardian/guui/dist/guui-svg-loader`
- `@guardian/guui/dist/guui-css-loader`
- `@guardian/guui`

The latter exposes `h` (for the purpose of providing the module during transpilation, above), `server` for server-rendering and `render` for client-rendering.

Initially, I wanted to expose `h` as a separate file to `server` and `render`. The problem I had with this approach is that Emotion is a singleton, whose state needs to be shared between our `h` wrapper (that invokes `emotion/css` and adds stylesheets to the Emotion singleton) and the `emotion-server` that reads stylesheets off the singleton when we extract the critical CSS.